tests/migrations/test_operations.py

```diff
@@ -5929,6 +5929,41 @@ def test_add_field_after_generated_field(self):
         self.assertEqual(pony_new.generated, 1)
         self.assertEqual(pony_new.static, 2)
 
+    def test_change_id_pk_to_uuid_pk(self):
+        app_label = "test_change_id_pk_to_uuid_pk"
+        project_state = self.set_up_test_model(app_label)
+        operation_1 = migrations.AddField(
+            "Pony",
+            "uuid",
+            models.UUIDField(primary_key=False),
+        )
+        operation_2 = migrations.RemoveField("Pony", "id")
+        operation_3 = migrations.RenameField("Pony", "uuid", "id")
```

Comment:
We might be able to leave off `operation_3` and `operation_4` - the code was recreating the ID field after `operation_2`, so maybe that's all we need for the test.

Reply:
The 3rd operation crashes so IMO it's still required.